app/views/users/show.html.erb

```diff
@@ -1,9 +1,32 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <%= flash[:message] %>
+
+<div>
+<h1><%= "#{@show_user.name.upcase }'S PROFILE'" %></h1>
+
+<% if current_user != @show_user &&  !current_user.friend?(@show_user)%>
+
+  <%= form_for @new_friendship do |f| %>
+    <%= hidden_field_tag "confirmer_id", @show_user.id %>
+
+    <div>
+      <%= f.submit 'Send Friend Request', class: "select_friend" %>
+    </div>
+  <% end %>
+
+<% end %>
+</div>
+<br>
+
+
+<% if current_user.friend?(@show_user) || current_user == @show_user %>
+
   <article class="timeline">
     <h3>Recent posts:</h3>
     <ul class="posts">
       <%= render @posts %>
     </ul>
   </article>
 </section>
+
+<% end %>
```

Comment:
- [ ] Instead of having the if/end statement create a helper method that has the if and renders a partial that contains the markup within the if/end statement instead of having the logic in the view